client/src/main.ts

```diff
@@ -1,8 +1,8 @@
-import { createApp } from 'vue'
-import App from './App.vue'
-import router from './router'
+import { createApp } from 'vue';
+import App from './App.vue';
+import router from './router';
 
 /**
  * Starts up the application.
  */
-createApp(App).use(router).mount('#app')
+const app = createApp(App).use(router).mount('#app');
```

Comment:
How come this variable?

Reply:
Was playing around and forgot to delete - nice catch!